test/Microsoft.Framework.PackageManager.FunctionalTests/Microsoft.Framework.PackageManager.FunctionalTests.xproj

```diff
@@ -13,8 +13,5 @@
   <PropertyGroup>
     <SchemaVersion>2.0</SchemaVersion>
   </PropertyGroup>
-  <ItemGroup>
-    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
-  </ItemGroup>
```

Comment:
Don't remove this. It's a test service reference and helps VS load the solution faster.